modules/installation-uninstall-clouds.adoc

```diff
@@ -52,10 +58,16 @@ ifdef::ibm-cloud[]
 * You have installed the IBM Cloud CLI and installed or updated the VPC infrastructure service plugin. For more information see "Prerequisites" in the link:https://cloud.ibm.com/docs/vpc?topic=vpc-infrastructure-cli-plugin-vpc-reference&interface=ui#cli-ref-prereqs[IBM Cloud VPC CLI documentation].
 endif::ibm-cloud[]
 
+ifdef::azure,ash[]
+While you can uninstall the cluster using the copy of the installation program that was used to deploy it, using version 4.13 or later is recommended.
```

Comment:
do you think we need {product-title} 4.13 or later at least once?

Reply:
Not necessarily @mburke5678. Referring to the installation program, so I believe it is implied and OK.

Reply:
I take it back. I see what you mean. I will add it to the first instance.